### ISSUE TITLE
[Bug]:  MO reboot failure with error

### ISSUE THREAD

**Issue by dengn** [matrixorigin/matrixone#5876]
### Is there an existing issue for the same bug?

- [X] I have checked the existing issues.

### Environment

```Markdown
- Version or commit-id (e.g. v0.1.0 or 8b23a93):2bfccbb1ede31f9f7ea356d253336dfce53c36b2
- Hardware parameters:
- OS type:
- Others:
```

### Actual Behavior

MO reboot failure and keep getting error messages:

<img width="1347" alt="image" src="https://user-images.githubusercontent.com/4965857/197438920-a6211a33-76ca-47e5-8339-2031ec780252.png">

**Comment by volgariver6** [comment]
resovled by #6012

**Comment by aressu1985** [comment]
fixed:1feeb04254c6fdab0ae0061ecc32349136d6ee10

no this panic any more, but still can not restart ,traced by #6284
